common/app/app/FrontendApplicationLoader.scala

```diff
@@ -53,13 +53,11 @@ trait FrontendComponents
   lazy val guardianConf = new GuardianConfiguration
   lazy val mode = environment.mode
 
-  // this is a workaround to make wsapi and the actorsystem available to the injector.
-  // I'm forced to do that as we still use Ws.url and Akka.system(app) *everywhere*, and both directly get the reference from the injector
-  override lazy val injector: Injector = new SimpleInjector(NewInstanceInjector) + router + crypto + httpConfiguration + wsApi + actorSystem + csrfConfig
+  override lazy val injector: Injector = new SimpleInjector(NewInstanceInjector) + router + crypto + httpConfiguration + csrfConfig
```

Comment:
Do we even need this? Is the default `BuiltInComponents` injector not providing everything we need already?